conjure/src/main/java/com/palantir/conjure/cli/ConjureCli.java

```diff
@@ -60,8 +67,15 @@ public void run() {
                 index = "1")
         private String output;
 
+        @CommandLine.Unmatched
+        @Nullable
+        private List<String> unmatchedOptions;
+
         @Override
         public void run() {
+            if (unmatchedOptions != null && !unmatchedOptions.isEmpty()) {
+                log.warn("Ignoring unknown options", SafeArg.of("unknown options", unmatchedOptions));
```

Comment:
Do we actually want `System.err.println` here? Using a logger for CLI user interaction can be problematic, for example slf4j-simple won't print the args because we don't have substitution braces.
If slf4j-simple is replaced with something else, it won't be obvious that this doesn't provide user feedback.

Reply:
haha I had originally gone with that then switched over to logging. I'll revert